### PR TITLE
Create a friendlier system of project documentation

### PR DESCRIPTION
**Pull request by seanbudd**
### Summary of the issue:

There are a few issues with the current project documentation:
* The README is quite long, and focused largely on configuring the developer environment.
Instead, it should be a homepage for the various aspects of the project. Detailed information should be split into their own documents.
* `contributing.md` is quite long and should be split similarly into the ways one can contribute to NVDA.
* There is a lot of missing information in terms of how to get started with various ways of contributing.

### Description of user facing changes

`README.md` is now split into
- Community information
  - support / getting help
  - getting add-ons
  - Issue reporting
  - General community information

`Contributing.md` is now split into
  - testing
  - translating
  - Issue triage
  - Development
    - Design
  - add-on dev
    - add-on distribution

Guides have been updated or added when missing.

### Known issues with pull request:
None

### Change log entries:
For Developers
The NVDA project documentation has been updated...

### CODE REVIEW

**Comment by seanbudd** [comment]
this is now ready for review

File: projectDocs/testing/contributing.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+## Manual testing
+
+Testing alpha / beta / and release candidates help to ensure the quality of the NVDA.
```
Comment (michaelDCurran) [inline projectDocs/testing/contributing.md:4]:
```suggestion
Testing alpha / beta / and release candidates help to ensure the quality of the NVDA project.
```

File: projectDocs/testing/contributing.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+
+## Manual testing
+
+Testing alpha / beta / and release candidates help to ensure the quality of the NVDA.
+You can the latest alpha snapshots [here](https://www.nvaccess.org/files/nvda/snapshots/).
```
Comment (michaelDCurran) [inline projectDocs/testing/contributing.md:5]:
```suggestion
You can find the latest alpha snapshots [here](https://www.nvaccess.org/files/nvda/snapshots/).
```

**Review by michaelDCurran — CHANGES_REQUESTED** [review]

**Comment by dpy013** [comment]
Recommended Use[gpl-2.0.md](https://www.gnu.org/licenses/old-licenses/gpl-2.0.md)
update
copying.txt

**Comment by seanbudd** [comment]
Could you please explain a bit more @dpy013 ? Note that this isn't a PR to update our licence. If you'd like to discuss our licence, I suggest opening a separate issue or discussion

**Review by Qchristensen — APPROVED** [review]
Great job on this, thank you.

File: projectDocs/community/readme.md [hunk 3]
```diff
@@ -15,19 +16,18 @@ This includes information on reporting issues, triaging issues, testing, transla
...
 * [Key Commands](https://www.nvaccess.org/files/nvda/documentation/keyCommands.html)
 * [Changes in the latest release](https://www.nvaccess.org/files/nvda/documentation/changes.html)
-* Further documentation is available in the NVDA repository's [Wiki](https://github.com/nvaccess/nvda/wiki), and in the [Community Wiki](https://github.com/nvaccess/nvda-community/wiki)
 
 ### Add-ons
 * [The NVDA Add-on store](https://www.nvaccess.org/files/nvda/documentation/userGuide.html#AddonsManager)
 * [Community add-ons website](https://addons.nvda-project.org/)
 
 ### Communication channels
+* NV Access has several ways you can contact us and keep up to date on the latest of NVDA.
```
Comment (seanbudd) [inline projectDocs/community/readme.md:25]: Fix typo
```suggestion
* NV Access has several ways you can contact us and keep up to date on the latest on NVDA.
```

**Comment by seanbudd** [comment]
Community feedback would be great here!
This is a big change, so feedback on structure as well as content would be welcome.
We'd like to make the content as accessible, logically compiled and easy to navigate as possible.

It might be easier to review the documents directly:
- [the new readme](https://github.com/nvaccess/nvda/blob/splitReadme/readme.md)
- [the new contributing file](https://github.com/nvaccess/nvda/blob/splitReadme/.github/CONTRIBUTING.md)
- [the new project docs folder](https://github.com/nvaccess/nvda/tree/splitReadme/projectDocs)

We've also updated the [wiki home page](https://github.com/nvaccess/nvda/wiki) to include a clearer contents index and migrated content from the [other wiki repository `nvda-community`](https://github.com/nvaccess/nvda-community/wiki)

**Comment by nvdaes** [comment]
In [Community readme](https://github.com/nvaccess/nvda/blob/splitReadme/projectDocs/community/readme.md), I wonder if the community add-ons website should be mentioned in addition to  the store:

- Personally, I'm not sure about if it's well maintained.
- - Imo, downloading add-ons from the store itself is more secure than installing them from external sources when possible, since checsum is used to ensure that releases are inmutable and that, when an add-on installed, the manifest contains NVDA API versions.
Then, if the community website is mentioned, I suggest to add a brief note about differences between the store and the website, for example: add-on store (recommended) or something.

**Comment by seanbudd** [comment]
The community add-ons website is still being supported by NV Access for now, and is the source of some add-ons that aren't available from the add-on store.

I agree that it's probably worth a note that this website is towards end of life and the add-on store is preferred

File: projectDocs/community/readme.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+### Communication channels
+* NV Access has several ways you can contact us and keep up to date on the latest on NVDA.
+See the [NV Access](https://www.nvaccess.org/contact-us/) website for our social media accounts and other contact information.
+* [NVDA Users Mailing List](https://nvda.groups.io/g/nvda)
+* [NVDA Developers Mailing List](https://groups.io/g/nvda-devel)
+* [NVDA Translators list](https://groups.io/g/nvda-translations)
+* [NVDA Add-ons Mailing List](https://groups.io/g/nvda-addons)
+* [NVDA Add-on API Mailing List](https://groups.google.com/a/nvaccess.org/g/nvda-api)
+* [NVDA Community Gitter channel](https://gitter.im/nvaccess/NVDA) (Instant Messaging)
+* [Other sources including groups and profiles on social media channels, language-specific websites and mailing lists etc.](https://github.com/nvaccess/nvda/wiki/Connect)
```
Comment (nvdaes) [inline projectDocs/community/readme.md:34]:
```suggestion
#### Managed by the community, not supported by NV Access ####
* [Other sources including groups and profiles on social media channels, language-specific websites and mailing lists etc.](https://github.com/nvaccess/nvda/wiki/Connect)

```

**Review by nvdaes — CHANGES_REQUESTED** [review]

**Comment by nvdaes** [comment]
Imo, maybe useful to inform what channels are supported by NV Access. Sometimes we, community members, may say that our website, mailing list or whatever resource is official, without clarifying the meaning of the "official" word. So I sugest to inform people what channels are supported by NV Access.

**Comment by seanbudd** [comment]
@nvdaes -I've added some subheaders to indicate this

**Pull request merged**